Vagrantfile

```diff
@@ -51,7 +51,8 @@ ansible_provision = proc do |ansible|
     fsid: '4a158d27-f750-41d5-9e7f-26ce4c9d2d45',
     monitor_secret: 'AQAWqilTCDh7CBAAawXt6kyTgLFCxSvJhTEmuw==',
     journal_size: 100,
-    monitor_interface: "enp0s8",
+    control_interface: "enp0s8",
+    netplugin_if: "enp0s8",
```

Comment:
@erikh do you want to set variable now or wait until we bring back `contiv_network` in volplugin host-group?